cvpy/tests/test_cvat_project.py

```diff
@@ -52,26 +58,96 @@ def test_cvat_project_invalid_user_password(self):
         except Exception as e:
             self.assertEqual(str(e), "Unable to log in with provided credentials.")
 
-    # Add a task to a project
-    def test_cvat_project_add_task(self):
```

Comment:
I've removed this test because it does not add anything beyond what test_cvat_project_post_images already tests. Due to the change in the constructor (required ImageTable, Project, and all ID's removed) there isn't anything unique to this test.